<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwidget</class>
 <widget class="QWidget" name="mainwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>苏嵌网课平台</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 15px</string>
  </property>
  <widget class="QLabel" name="lblTtitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>1001</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>苏嵌网课平台</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>0</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(18, 218, 218);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QFrame" name="frmPlay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>1001</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>个人信息</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/VlcPlayer/D:/苏嵌合作/图片/个人信息.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="lstvPlayList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>51</y>
     <width>81</width>
     <height>661</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>播放列表</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 147, 147)</string>
   </property>
  </widget>
  <widget class="CustomSlider" name="tbPlayTack">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>690</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRate">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>688</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlayTime">
   <property name="geometry">
    <rect>
     <x>938</x>
     <y>690</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0:0:0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTotalTime">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>690</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0:0:0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>690</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(85, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>上传视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="prbFileUp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QSlider</extends>
   <header location="global">customslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mainwidget</receiver>
   <slot>Close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1041</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>995</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRate</sender>
   <signal>clicked()</signal>
   <receiver>mainwidget</receiver>
   <slot>RateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>963</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>963</x>
     <y>803</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>mainwidget</receiver>
   <slot>UPFileClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>759</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCut</sender>
   <signal>clicked()</signal>
   <receiver>mainwidget</receiver>
   <slot>CutUpLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>745</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Close()</slot>
  <slot>RateClick()</slot>
  <slot>UPFileClick()</slot>
  <slot>CutUpLoad()</slot>
 </slots>
</ui>
